app/Models/BaseAlbumImpl.php

```diff
@@ -190,6 +203,7 @@ class BaseAlbumImpl extends Model implements HasRandomID
 	protected $appends = [
 		'has_password',
 		'sorting',
+		'policies',
```

Comment:
> I wonder whether we should hide the direct attributes grants_download, etc., too, in order to avoid redundancy.

In theory this is already the case:
```
	/**
	 * @var array<int,string> The list of attributes which exist as columns of the DB
	 *                        relation but shall not be serialized to JSON
	 */
	protected $hidden = [
		HasRandomID::LEGACY_ID_NAME,
		'owner_id',
		'owner',
		'password',
		'sorting_col',   // serialize DTO `order` instead
		'sorting_order', // serialize DTO `order` instead

		// Security attributes are hidden because provided by the DTO AlbumProtectionPolicy
		'is_public',
		'is_link_required',

		// Permissions are hidden because they will eventually be replaced by an external table
		// and are provided by the AlbumRightsDTO
		'grants_download',
		'grants_full_photo_access',
	];
```

Reply:
Sorry, my fault. I must have missed that part.